spipe/main.c

```diff
@@ -250,7 +250,7 @@ main(int argc, char * argv[])
 	if ((conn_cookie = proto_conn_create(s[1], sas_t, 0, opt_f, opt_g,
 	    opt_j, K, opt_o, callback_conndied, &ET)) == NULL) {
 		warnp("Could not set up connection");
-		goto err2;
+		goto err3;
```

Comment:
If we fail at this point, isn't `s[1]` leaked?  But in the error path we're only closing `s[0]`...

Reply:
Fixed in a new commit, because technically it's not part of "don't close... in pushbits".